static/src/stylesheets/base/_inline-svgs.scss

```diff
@@ -6,6 +6,15 @@
     }
 }
 
+.inline-icon--light-grey {
+    fill: colour(neutral-3);
+}
+
+.inline-close.small {
```

Comment:
Yes that makes sense. This is a different style guide to what I am used to so thanks so much for checking. I will submit a new PR with this change.